module/FileNVRAM.c

```diff
@@ -359,7 +388,18 @@ static void readplist()
                 
                 if (plistSize && read(fh, plistBase, plistSize) == plistSize)
                 {
-                    XMLParseFile( plistBase, &gPListData );
+                    // cleaning nvram.plist from \n and \t.
+                    // data tag have those bytes after the serialization:
+                    //
+                    //  <key>SystemAudioVolume</key>
+                    //  <data>
+                    //      fw==
+                    //  </data>
+                    //
+                    // but I want it to be: <data>fw==</data>
+                    //
+                    // so some garbage ends up in /chosen/nvram. just remove them!
+                    XMLParseFile(strdelchar(strdelchar(plistBase, '\n'), '\t'), &gPListData);
```

Comment:
Should I reimplement it? I don't had the time to look at chameleon, but I think that I can do that in the BASE64Decode() function (which in truth should fail if there are foreign characters). Or in ParseTagData()